src/test/java/org/folio/rest/api/HoldingsStorageTest.java

```diff
@@ -813,6 +829,443 @@ public void cannotCreateHoldingWithoutPermanentLocation() throws Exception {
       is("permanentLocationId"));
   }
 
+  @Test
+  public void canCreateAHoldingsWhenHRIDNotSupplied()
```

Comment:
What is the difference between this test and `canCreateAHolding`? Could they be combined?

Reply:
I don't like modifying existing tests with new functionality. I'm afraid I'll mask regression by changing existing behavior expectations. I understand that leads to a lot of tests that do pretty much the same thing, but with different expectations. I can combine the tests.

Reply:
> I don't like modifying existing tests with new functionality. I'm afraid I'll mask regression by changing existing behavior expectations. I understand that leads to a lot of tests that do pretty much the same thing, but with different expectations. I can combine the tests.

I sympathise with that approach. And if these were lightweight unit tests I'd be in total agreement, because they better reflect intent. However, with hefty API tests there seems to be more of a compromise.

Hopefully in this case, the only expectation being changed is that the `hrid` has a generated value?